sklearn/metrics/classification.py

```diff
@@ -1389,6 +1388,10 @@ def balanced_accuracy_score(y_true, y_pred, sample_weight=None):
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    adjusted : bool, default=False
+        When true, the result is adjusted for chance, so that random
+        performance would score 0, and perfect performance scores 1.
+
     Returns
     -------
     balanced_accuracy : float.
```

Comment:
We might change the sensitivity/specificity explanation.

Reply:
Good catch